Formula/crystal-lang.rb

```diff
@@ -57,13 +56,13 @@ def install
     ENV["CRYSTAL_CONFIG_PATH"] = prefix/"src:lib"
     ENV.append_path "PATH", "boot/bin"
 
-    if build.with? "release"
-      system "make", "crystal", "release=true"
-    else
-      system "make", "deps"
-      (buildpath/".build").mkpath
-      system "bin/crystal", "build", "-o", "-D", "without_openssl", "-D", "without_zlib", ".build/crystal", "src/compiler/crystal.cr"
-    end
+    system "make", "deps"
+    (buildpath/".build").mkpath
+
+    command = ["bin/crystal", "build", "-D", "without_openssl", "-D", "without_zlib", "-o", ".build/crystal", "src/compiler/crystal.cr"]
+    command.concat ["--release", "--no-debug"] if build.with? "release"
```

Comment:
Since `--no-debug` was needed for a successful build, doesn't this imply that the `--without-release` option will be broken?

Reply:
@ilovezfs no, `--no-debug` is only an issue on release builds. LLVM only has this bug when running optimizations.